presto-base-arrow-flight/src/test/java/com/facebook/plugin/arrow/TestingArrowMetadata.java

```diff
@@ -150,8 +145,14 @@ protected String getDataSourceSpecificTableName(ArrowFlightConfig config, String
     }
 
     @Override
-    protected ArrowFlightRequest getArrowFlightRequest(ArrowFlightConfig config, Optional<String> query, String schema, String table)
+    protected FlightDescriptor getFlightDescriptor(ArrowFlightConfig config, Optional<String> query, String schema, String table)
     {
-        return new TestingArrowFlightRequest(config, testconfig, schema, table, query, nodeManager.getWorkerNodes().size());
+        TestingArrowFlightRequest request = new TestingArrowFlightRequest(config, testconfig, schema, table, query, nodeManager.getWorkerNodes().size());
+        return FlightDescriptor.command(request.getCommand());
+    }
+
+    private TestingArrowFlightRequest getArrowFlightRequest(ArrowFlightConfig config, String schema)
```

Comment:
`config` is available in class member, no need to pass in method parameters

Reply:
done

Reply:
Is `config` still needed in this method's parameters

Reply:
removed